superset-frontend/src/explore/components/controls/MetricControl/MetricDefinitionOption.test.jsx

```diff
@@ -16,38 +16,38 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import configureStore from 'redux-mock-store';
-import { shallow } from 'enzyme';
+import { render, screen, waitFor } from 'spec/helpers/testing-library';
 
 import MetricDefinitionOption from 'src/explore/components/controls/MetricControl/MetricDefinitionOption';
-import AggregateOption from 'src/explore/components/controls/MetricControl/AggregateOption';
-import {
-  StyledMetricOption,
-  StyledColumnOption,
-} from 'src/explore/components/optionRenderers';
+import userEvent from '@testing-library/user-event';
 
-describe('MetricDefinitionOption', () => {
-  const mockStore = configureStore([]);
-  const store = mockStore({});
-
-  function setup(props) {
-    return shallow(<MetricDefinitionOption store={store} {...props} />).dive();
-  }
-
-  it('renders a StyledMetricOption given a saved metric', () => {
-    const wrapper = setup({
-      option: { metric_name: 'a_saved_metric', expression: 'COUNT(*)' },
+const renderMetricDefinitionOption = props => {
+  waitFor(() => {
```

Comment:
This `waitFor` is not necessary as you are already waiting for the elements in your tests.

Reply:
Removed in HEAD :D